src/terraform/index_test.go

```diff
@@ -69,7 +69,7 @@ provider "aws" {
 }
 
 module "aws" {
-  source = "./aws"
+  source = "git@github.com:Originate/exosphere.git//src//terraform//modules//aws?ref=8786f912"
```

Comment:
`//` is the Terraform syntax to specify subdirs. The `ref=8786f912` at the end specifies the commit.